src/analysis/properties.rs

```diff
@@ -128,19 +127,25 @@ fn analyze_property(
         .min()
         .or(prop.version)
         .or_else(|| Some(env.config.min_cfg_version));
+    let generate = configured_properties
+        .iter()
+        .filter_map(|f| f.generate)
+        .next()
+        .unwrap_or_else(PropertyGenerateFlags::all);
     let name_for_func = nameutil::signal_to_snake(&name);
     let var_name = nameutil::mangle_keywords(&*name_for_func).into_owned();
     let get_func_name = format!("get_property_{}", name_for_func);
     let set_func_name = format!("set_property_{}", name_for_func);
     let check_get_func_name = format!("get_{}", name_for_func);
     let check_set_func_name = format!("set_{}", name_for_func);
 
-    let mut readable = prop.readable;
+    let mut readable = prop.readable && generate.contains(PropertyGenerateFlags::GET);
```

Comment:
It should probably warn if `GET` is configured but the property is write-only (and same for writable)

Reply:
Good idea